tests/ui/cfg/true-false.rs

```diff
@@ -1,7 +1,6 @@
 //@ run-pass
 
 #![feature(link_cfg)]
-#![feature(cfg_boolean_literals)]
```

Comment:
Note: not a blocker for this stabilization, but we should reject revision names `true` and `false` in compiletest as well, because they will try to set `--cfg=true` and `--cfg=false`.

Reply:
https://github.com/rust-lang/rust/issues/138663